src/components/shared/Validations.tsx

```diff
@@ -161,8 +174,17 @@ export function validationCount(schemaNode: RegularNode) {
 export function getValidationsFromSchema(schemaNode: RegularNode) {
   return {
     ...(schemaNode.enum !== null ? { enum: schemaNode.enum } : null),
-    ...('annotations' in schemaNode && schemaNode.annotations.default
-      ? { default: schemaNode.annotations.default }
+    ...('annotations' in schemaNode
+      ? {
+          ...(schemaNode.annotations.default ? { default: schemaNode.annotations.default } : null),
+          ...(schemaNode.annotations.examples ? { examples: schemaNode.annotations.examples } : null),
+        }
+      : null),
+    ...('fragment' in schemaNode
```

Comment:
We shouldn't be using fragment.
You can leave a todo note here (or even better create an issue in jst) and I'll get it done soon.

Reply:
Alright, I'll put a note here and create issue for JST

Reply:
JST bumped and fragment usage removed